src/themes/default/components/Carousel/_carousel.scss

```diff
@@ -5,17 +5,17 @@
     width: 100%;
 
     margin: {
-        top: $message-spacing-thick;
-        bottom: $message-spacing-thick;
+        bottom: $message-spacing-medium;
     }
 
     &-item {
         display: inline-block;
         vertical-align: top;
-        padding: {
-            left: $message-spacing-thick;
-            right: $message-spacing-thick;
-        }
+        //TODO: Removed this, because arrows were removed, thus padding for them should be removed.
```

Comment:
which arrows were removed ?

Reply:
The left arrow and the right arrow alongside the carousel.